src/index.html

```diff
@@ -11,12 +11,16 @@
     <link rel="stylesheet" href="table.css" />
     <script>
       function initSasJs() {
-        sasJs = new SASjs.default({
-          appLoc: "/Public/app/fileuploader",
-          serverType: "SASVIYA",
+        sasjs = new SASjs.default({
+          appLoc: '/Public/app/saad/fileuploader',
+          serverType: 'SASVIYA',
+          serverUrl: 'https://sas.analytium.co.uk',
```

Comment:
need to leave this undefined so it can be dynamically obtained

Reply:
removed